### ISSUE TITLE
about scale_lr

### ISSUE THREAD

**Issue by 704051419** [Jia-Research-Lab/PFENet#17]
In the experiment setting of PASCAL-5i, the initial lr is setting to 0.0025. But i saw that there was a scaler_lr in the poly_learning_rate(). And the default value is 10. So the initial lr is actually 0.025. right?

**Comment by tianzhuotao** [comment]
@704051419 In train.py, the index has been set as -1 which means no learning rate value is scaled by 10. Therefore the initial lr is still 0.0025 for all learnable parameters.

Details can be found at:
https://github.com/Jia-Research-Lab/PFENet/blob/master/train.py#L255
